app/controllers/admin/student_accounts_controller.rb

```diff
@@ -0,0 +1,37 @@
+require 'date'
+
+module Admin
+    class StudentAccountsController < BaseController
+      layout 'admin'
+  
+      def actions
+      end
+  
+      def get_number_of_accounts
+        days_num = get_number_of_days
+        accounts_num = query(days_num)
+        flash[:notice] = "The number of student acccounts created since July 1 is " + accounts_num.to_s
```

Comment:
Please use string interpolation like such: `"The number of student acccounts created since July 1st 2019 is #{result}"`

Reply:
Requested change has been made